projects/plugins/jetpack/class.jetpack.php

```diff
@@ -1007,64 +990,6 @@ public function allow_wpcom_domain( $domains ) {
 		return array_unique( $domains );
 	}
 
-	/**
-	 * Redirect edit post links to Calypso.
-	 *
-	 * @param string $default_url Post edit URL.
-	 * @param int    $post_id Post ID.
-	 *
-	 * @return string
-	 */
-	public function point_edit_post_links_to_calypso( $default_url, $post_id ) {
```

Comment:
Since those are public methods, could we keep them for a couple of releases and mark them as deprecated with `_deprecated_function( __METHOD__`?

Reply:
yes

Reply:
Addressed in 3c729cf27878a8c90a7516b774191fbfe7fa6d40 and 3eb3b7a46ffedcbad69afe451266a08bb8b42188